IOPool/Input/test/testSchemaEvolution.sh

```diff
@@ -102,4 +102,6 @@ inputfile=$(edmFileInPath IOPool/Input/data/$file) || die "Failure edmFileInPath
 #root.exe -b -l -q file:$inputfile "${LOCAL_TEST_DIR}/testForStreamerInfo.C(gFile)" | sort -u | grep Missing > testForStreamerInfo2.log
 #grep "Missing" testForStreamerInfo2.log && die "Missing nested streamer info" 1
 
+cmsRun ${LOCAL_TEST_DIR}/SchemaEvolution_test_read_cfg.py --inputFile $inputfile --enableStreamerInfosFix || die "Failed to read old file $file with fix" $?
```

Comment:
Maybe the comment above could be rephrased now e.g. along "the test would fail without the `--enableStreamerInfosFix`"?